src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -137,7 +137,7 @@ Read more about .NET Core CLI Tools telemetry: https://aka.ms/dotnet-cli-telemet
 Explore documentation: https://aka.ms/dotnet-docs
 Report issues and find source on GitHub: https://github.com/dotnet/core
 Find out what's new: https://aka.ms/dotnet-whats-new
-Learn about the installed HTTPS developer cert: https://aka.ms/aspnet-core-https
+Configure HTTPS development certificates: https://aka.ms/aspnet-core-https
```

Comment:
Should this be singular instead of plural? Do you ever have more than one dev cert?

Reply:
Don't use plural. We've already go a question asking how to create multiple certs. Plural will open a floodgate.